openshift/all-jobs-template.yaml

```diff
@@ -0,0 +1,561 @@
+                - name: THOTH_LOG_SOLVER
+                  value: "${THOTH_LOG_SOLVER}"
+                - name: THOTH_SOLVER_NO_TRANSITIVE
+                  value: "${THOTH_SOLVER_NO_TRANSITIVE}"
+                - name: THOTH_SOLVER_PACKAGES
+                  value: "${THOTH_SOLVER_PACKAGES}"
+                - name: THOTH_SOLVER_OUTPUT
+                  value: "${THOTH_SOLVER_OUTPUT}"
+                - name: THOTH_SOLVER_INDEXES
+                  value: "${THOTH_SOLVER_INDEXES}"
+                - name: THOTH_SOLVER_VIRTUALENV
+                  value: "/home/solver/venv"
+                - name: SENTRY_DSN
+                  valueFrom:
+                    secretKeyRef:
+                      name: thoth
+                      key: sentry-dsn
+              image: "${IMAGE_STREAM_REGISTRY}/${IMAGE_STREAM_NAMESPACE}/solver-fedora-29-py36:${IMAGE_STREAM_TAG}"
+              livenessProbe:
+                failureThreshold: 1
+                initialDelaySeconds: 21600
+                periodSeconds: 10
+                tcpSocket:
+                  port: 80
+              resources:
+                limits:
+                  cpu: 1
+                  memory: 1Gi
+                requests:
+                  cpu: 1
+                  memory: 1Gi
+
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: "${THOTH_SOLVER_JOB_ID}"
+      annotations:
+        thoth-station.ninja/template-version: 0.9.1
+      labels:
+        app: thoth
+        component: solver
+        solver-type: solver-fedora-28-py36
+        operator: graph-sync
+        task: solver
+        mark: cleanup
+    spec:
+      template:
+        metadata:
+          labels:
+            app: thoth
+            component: solver
+            solver-type: solver-fedora-28-py36
+            mark: cleanup
+            task: solver
+            ttl: 2h
+        spec:
+          restartPolicy: OnFailure
+          containers:
+            - name: solver
+              env:
+                - name: THOTH_SOLVER
+                  value: solver-fedora-28-py36
+                - name: THOTH_LOG_SOLVER
+                  value: "${THOTH_LOG_SOLVER}"
+                - name: THOTH_SOLVER_NO_TRANSITIVE
+                  value: "${THOTH_SOLVER_NO_TRANSITIVE}"
+                - name: THOTH_SOLVER_PACKAGES
+                  value: "${THOTH_SOLVER_PACKAGES}"
+                - name: THOTH_SOLVER_OUTPUT
+                  value: "${THOTH_SOLVER_OUTPUT}"
+                - name: THOTH_SOLVER_INDEXES
+                  value: "${THOTH_SOLVER_INDEXES}"
+                - name: THOTH_SOLVER_VIRTUALENV
+                  value: "/home/solver/venv"
+                - name: SENTRY_DSN
+                  valueFrom:
+                    secretKeyRef:
+                      name: thoth
+                      key: sentry-dsn
+              image: "${IMAGE_STREAM_REGISTRY}/${IMAGE_STREAM_NAMESPACE}/solver-fedora-28-py36:${IMAGE_STREAM_TAG}"
```

Comment:
This one is too old. Let's drop it. I'm also not that confident with `f29-py*`.

@goern Comments/thoughts on supported OS releases?

Reply:
I will drop them :+1: , Thanks for feedback.